packages/react-core/src/demos/ComposableMenu/examples/ComposableContextSelector.tsx

```diff
@@ -118,8 +118,8 @@ export const ComposableContextSelector: React.FunctionComponent = () => {
     </MenuToggle>
   );
 
-  const onSelect = (ev: React.MouseEvent<Element, MouseEvent>, itemId: string) => {
-    setSelected(itemId);
+  const onSelect = (ev: React.MouseEvent<Element, MouseEvent>, itemId: string | number) => {
+    setSelected(itemId.toString());
```

Comment:
fixed as i came across a red squiggly on the `Menu.onSelect` type mismatch (line 150)